zbus/src/abstractions/timeout.rs

```diff
@@ -0,0 +1,36 @@
+use crate::{Error, Result};
+use std::{future::Future, io::ErrorKind, time::Duration};
+
+/// Awaits a future with a provided timeout
```

Comment:
I wasn't going to mention this but since you're going to modify the PR again anyway, please add the missing `.` at the end. (Same for the duplicate below).